examples/canon.js

```diff
@@ -19,9 +18,21 @@ module.exports = {
       }
     }
   },
-  reduxMiddleware(applyMiddleware, canonMiddleware) {
-    const message = "reduxMiddleware from canon.js";
-    console.log(message);
-    return applyMiddleware(...exampleReduxMiddleware(canonMiddleware));
-  }
+  db: [
+    {
+      connection: env.CANON_SERVER_DBCONNECTION ||
+        `postgresql://${env.CANON_DB_USER}:${env.CANON_DB_PASS}@${env.CANON_DB_HOST || "localhost"}:${env.CANON_DB_PORT || 5432}/${env.CANON_DB_NAME}`,
+      tables: [
+        coreModelPaths.users,
+        require("./db/testTable")
+      ]
+    },
+    {
+      host: env.CANON_DB_HOST || "localhost",
+      name: env.CANON_DB_NAME,
+      user: env.CANON_DB_USER,
+      pass: env.CANON_DB_PW,
+      tables: Object.values(cmsModelPaths)
+    }
+  ]
```

Comment:
This worried me at first, because I thought that we were going to have to require all user sites to create `models.js` type files re-listing the tables, but now I see what's going on and this really fantastic! 🍻 

Similar to my comment about the redux changes, can you add documentation to the core README file on how to set up database connections using `canon.js`, and then write some migration notes in this PR to be used in the release notes?

Reply:
To migrate a database, the DB environment variables the app was using must be setup in a new connection detail object in the `canon.js` file, following the instructions on the documentation.